pyproject.toml

```diff
@@ -31,21 +31,22 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-    "pip-tools~=7.4.1",
-    "setuptools-git-versioning>=2.0, <3",
-    "black~=24.8.0",
-    "flake8~=7.1.1",
-    "Flake8-pyproject~=1.2.3",
-    "flake8-docstrings~=1.7.0",
-    "flake8-quotes~=3.4.0",
-    "flake8-pytest-style~=2.0.0",
-    "flake8-bugbear~=24.8.19",
-    "flake8-mock~=0.4",
-    "flake8-tuple~=0.4.1",
-    "pytest~=7.4.4",
-    "pytest-cov~=5.0.0",
-    "mypy~=1.11.2",
-    "build~=1.0.3",
+    "setuptools ~= 75.6.0",
```

Comment:
This update should prevent you all from receiving anymore e-mails from pypi when performing a release. They are moving to a new naming standard that has been fixed in setuptools >= 69.3.0